tests/operator/Makefile

```diff
@@ -4,3 +4,17 @@ include ${PROJECT_ROOT}/hack/help.mk
 .PHONY: test
 test: ## Run integration test.
 	go run main.go
+
+.PHONY: cluster-info
```

Comment:
You can try to look use this script: https://github.com/kyma-project/serverless/blob/main/tests/serverless/collect_logs.sh